packages/mui-material/src/Select/Select.test.js

```diff
@@ -482,7 +482,7 @@ describe('<Select />', () => {
       const { getByRole } = render(<Select open value="" />);
       const listboxId = getByRole('listbox').id;
 
-      expect(getByRole('combobox', { hidden: true })).to.have.attribute('aria-controls', listboxId);
```

Comment:
while I was running the test locally, the test failed without this option.

Reply:
Yeah, just realized this 😅 
Reverted